controllers/news-controllers.js

```diff
@@ -36,3 +36,22 @@ exports.getAllArticles = (request, response, next) => {
     .catch(next)
 }
 
+exports.getCommentsByArticleId = (request, response, next) => {
+    const { article_id } = request.params;
+    fetchArticleById(article_id)
+    .then((article) => {
+        if (!article) {
```

Comment:
Better to put any custom errors in your model function using Promise.reject() refer to the errorhandling notes